doc/contributing.md

```diff
@@ -485,8 +485,8 @@ For consistency, we also use UNIX-style line endings (`\n`) and file permission
 644 (`-rw-r--r--`) throughout the whole project.
 Don't worry if you forget to do it. Our continuous integration systems will
 warn us and you can make a new commit with the formatted code.
-Even better, you can just write `/format` in the first line of any comment in a
-pull request to lint the code automatically.
+Even better, you can just write `/format` and/or `pre-commit.ci autofix` in the first
```

Comment:
Changes in this file can be reverted.

Reply:
Done at 0f09fb605e4a4248c0ac0ca4766fb91299ddb6aa